scss/_navbar-offcanvas.scss

```diff
@@ -81,12 +81,14 @@
     }
   }
 
+  .nav {
+    max-height: 85vh;
+    flex-wrap: initial;
+  }
```

Comment:
Is this to address the overflow scroll issue? Could you share steps to reproduce the scrolling issue? And where is this class used within the `navbar-offcanvas` div (I think I'm missing something)?

Reply:
In AZ Barrio, the `<ul>` rendered in the main menu template has this class applied to it (see [themes/custom/az_barrio/templates/navigation/menu--main.html.twig#L33](https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/templates/navigation/menu--main.html.twig#L33)) and that template is used to render the main menu inside the `navigation_offcanvas` region in the page template when the `Use Navbar Off Canvas` theme setting is enabled (see [themes/custom/az_barrio/templates/page.html.twig#L138-L161](https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/templates/page.html.twig#L138-L161))

Reply:
After discussing with @joeparsons, he is going to create a separate PR to remove the `nav` class from `navbar-nav` since they should not be used together.

Reply:
Remove for now, address Barrio/Bootstrap discrepancy issue and see if there are scrolling issues with this ticket. https://github.com/az-digital/arizona-bootstrap/issues/213

Reply:
```
  .nav {
    max-height: 85vh;
    flex-wrap: initial;
  }